File: 프로그래머스/lv1/68935. 3진법 뒤집기/3진법 뒤집기.js
function solution(n) {
    let answer = 0;
    let bowl = n.toString(3);
    console.log(bowl + "  " + bowl.length);
    
    for(let i = bowl.length - 1; i >= 0; i--){
        console.log(bowl.substr(i, 1));
        answer += parseInt(bowl.substr(i, 1)) * Math.pow(3,i);
    }
    
    
    return answer;
}

File: 프로그래머스/lv1/68935. 3진법 뒤집기/3진법 뒤집기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        StringBuilder builder = new StringBuilder();
        
        // 3진수 변환 
        builder.append(Integer.toString(n, 3));
        // 값 반전
        builder.reverse();
        
        // 10진수 변환
        int a = builder.length()-1;
        for (int i=0;i< builder.length();i++){
            int before = Integer.parseInt(builder.substring(i, i + 1)); 
            answer += before * Math.pow(3,a);
            a--; 
        } 
        
        return answer;
    }
}